markdown/GameGuides/EldenRing.md

```diff
@@ -0,0 +1,98 @@
+  </div>
+  <input type="range" min="0" max="100" value="50" step="0.01" 
+    id="slider" class="slider__input" 
+    autocomplete="off" onwheel="this.blur()" 
+  />
+</div>
+<div class="figure"><p>Left: example of the composed ER 3:4 shot at resolution of 3037x4050 surrounded by the black bars in 16:10 monitor. Right: example in windows after the shot was taken. So you get a nice crisp 3:4 image result, no black bars.</p></div>
+
+This method comes with a penalty though, which is needing to compose a shot on high-resolution. For instance, if you align your shot in 16:10 and then switch back to 3:4, you will need to make some adjustments in the 3:4 aspect ratio.
+
+Having the beefier card will make these things significantly much easier of course (you can even play the game in 5k), but as an example, to give you a general idea, I have GTX 1060 6GB and I am still able to take high-resolution shots from this game. 
+
+Its a bit slower progress of course with the first method because sometimes, if not always, I tweak FoV while I am running the game at high DSR res because if your monitor has a 16:9 ratio, and you want to take the shot in 21:9, you will need to compose a bit on that DSR res and if you turned on Higher LODs from the tools along with ReShade it can be quite laggy. But there is also the other way of taking the high-res shots in this game.
+
+### Fullscreen Way
+@alert tip
+You can access the video settings even if your game is paused, while the free cam is on! This can be useful if you want to switch to different aspect ratios in fullscreen mode while not ruining your perfectly aligned shot. You just gotta do this with a keyboard and mouse if the camera control device is set to `Gamepad` in tools. If it's set to `KeyboardMouse`, then you can access the menu with the controller. If it's set to `Both` it will not work. Don't forget to turn the `HUD` back on otherwise you won't see the menu after pressing the `ESC`.
+@end
+
+Let's say that you are in 21:9, or 16:9, but you realised that 3:4 would suit you better for this moment, or 1:1, or you just want to experiment, so you press `ESC`, and navigate with a mouse (if the camera control device is set to `Gamepad` in tools) to video settings to pick the resolution of your choice and it will instantly switch to that DSR resolution! You can do this as many times as you want. Honestly, I never experienced crashes while doing this countless of times in one play session of 4 hours.
+
+It's really rare to happen, but keep in mind that this method sometimes won't work if you press the `Alt`+`Enter` or the `Windows` key for some reason or by accident. You will see the menu, but you won't be able to access it. So pay attention to that if you want to use this method straight away, or reset to refresh if the menu bug out, combine. But like I said it's rare, and it will only happen if you `Alt`+`Tab` too much while you are in a custom aspect ratio.
```

Comment:
Please rephrase this more generally so you avoid 'I' in the guide. (as it begs the question who is "I" )

Reply:
"It is I, Leclerc"! 😂  Almost perfect guide, right. I just had to complicate it in the end. Classic me

Reply:
Maybe I will delete the first paragraph or at least some of the text. I mean, I just literally repeated myself after the tip. But the reason why I do that is to achieve a friendly vibe for a reader, if I can explain it like that. You should see my facial expressions while writing that paragraph. I just want to explain why I did it like that. But yeah, I also thought about further corrections these days anyway. Just RE Village got me hooked up quite good these days :)